components/eam/src/chemistry/utils/tracer_data.F90

```diff
@@ -1876,6 +2026,18 @@ subroutine interpolate_trcdata( state, flds, file, pbuf2d )
                    call vert_interp_mixrat(ncol,file%nlev,pver,state(c)%pint, &
                         datain, data_out(:,:), &
                         file%p0,ps,file%hyai,file%hybi)
+                else if(file%linoz_v3 .or. file%linoz_v2) then
+                !!uci chemistry for linoz that better conserves mass
+                !!uci interpolation for 55 out of 57 variables in linoz_v3
+                !!excluding t_clim, o3col_clim
```

Comment:
Please confirm if this comment is update-to-date.

Reply:
Updated.